core/embed/trezorhal/stm32f4/flash.c

```diff
@@ -167,6 +167,50 @@ secbool flash_area_erase_bulk(const flash_area_t *area, int count,
   return sectrue;
 }
 
+secbool flash_area_erase_partial(const flash_area_t *area, uint32_t offset,
```

Comment:
this needs also unix implementation for bootloader emulator to work. (or alternatively flash_common implementation and only implement flash_erase_sector for both f4 and unix, in flash common there is also get_sector_and_offset function that can can be used for getting the sector index)

Reply:
unix implementation added 0636c0f50b2199f6f3f80782c82c3252a83f75ae.

Reply:
btw are you using the underscore in parameter name to indicate it is output? because in rust its used to indicate that the parameter is unused, and it might be confusing to mix those meanings

Reply:
Yes, it might be confusing. I've changed it. 8feb089e11a8b24c0df4775682fc3b4e237a0b11.

Reply:
it remained in the header file and the comment.

Reply:
fixed 1ff1235d27d1790071e05c3f1d49df41d998bc7e

Reply:
fixed b217e3c144650b89cfae5fe1a1929b6b7e015e3d

Reply:
I've fixed emulator build, see 9bb8f1df6934a387f5025c99e797305f7d7c1594. As we discussed, it's just temporary solution.